config/environments/test.rb

```diff
@@ -118,6 +118,10 @@
   PDF_STORAGE = "#{::Rails.root.to_s}/data/test/pdfs"
 
   ###################################################################
+  # Location of the public and private key for the git user on the system
+  GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.